src/ngraph/CMakeLists.txt

```diff
@@ -145,6 +145,7 @@ set (SRC
     op/binary_convolution.hpp
     op/broadcast.cpp
     op/broadcast_distributed.cpp
+    op/bucketize.cpp
```

Comment:
also add op/bucketize.hpp here as for the others. (I don't know why the hpp need to be added here though, but just to follow the same behavior).